apps/site/assets/ts/hooks/useGlxOpen.ts

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from "react";
+
+const glxStations = ["place-lech", "place-unsqu", "place-spmnl"];
+
+const useGlxOpen = (stationId: string): boolean => {
+  const [isGlxOpen, setIsGlxOpen] = useState(false);
+  useEffect(() => {
+    const glxOpen = document.querySelector(".glx-is-open");
+    if (glxOpen instanceof HTMLElement) {
+      setIsGlxOpen(glxOpen.dataset.open === "true");
+    }
+  }, []);
+
```

Comment:
Is `isGlxOpen` a variable that needs to be stateful? 🤔I am wondering what React is adding here as opposed to writing this as a simpler vanilla TypeScript helper function.

Reply:
removed hook entirely and just moved into the component.